Lib/enum.py

```diff
@@ -77,6 +77,9 @@ def __setitem__(self, key, value):
                     ):
                 raise ValueError('_names_ are reserved for future Enum use')
             if key == '_generate_next_value_':
+                # check if members already defined
+                if len(self._member_names) > len(dir(super)):
```

Comment:
`if self._member_names` is sufficient -- if any have been it will evaluate as *truthy*